Graph/readme.md

```diff
@@ -42,3 +42,6 @@ A graph is a pair (V, E), where V is a set of nodes, called vertices, and £ is
 - Kruskal Algorithm ----> [C++](/Code/C++/Kruskal.cpp) | [Java]() | [Python]()
 - Topological Sort ----> [C++](/Code/C++/Topological_sort.cpp) | [Java]() | [Python]()
```

Comment:
Why are you removing previous data?

Reply:
Sir I have created. a new Pr for this code with all the suggested changes .Since I told u that I am new in this So ,I thought I should create fresh from the start .Please have a look over it